app/javascript/components/util/DeleteDialog.js

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import { PropTypes } from 'prop-types';
+import { Button, Form, Modal } from 'react-bootstrap';
+
+class DeleteDialog extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      loading: false,
+      disabled: true,
+      showTextInput: false,
+    };
+  }
+
+  handleReasonChange = event => {
+    event.persist();
+    const showTextInput = event.target.value === 'Other';
+    this.setState({ disabled: false, showTextInput, [event.target.id]: event.target.value }, () => {
+      this.props.onChange(event);
+    });
+  };
+
+  delete = () => {
+    this.setState({ loading: true }, () => {
+      this.props.delete();
+    });
+  };
+
+  render() {
+    return (
+      <Modal size="lg" show centered onHide={this.props.toggle}>
+        <Modal.Header>
+          <Modal.Title>Delete {this.props.type}</Modal.Title>
+        </Modal.Header>
+        <Modal.Body>
+          <p>
+            Are you sure you want to delete this {this.props.type}?&nbsp; This action cannot be undone.&nbsp; For auditing purposes, this deletion will be
```

Comment:
Why the `&nbsp;` here? if you need, you can insert an empty space `{' '}` for line break issues? This currently is causing two spaces

https://user-images.githubusercontent.com/17532163/117021810-add6c380-acc5-11eb-82cf-ded2be1d183e.mov

Reply:
fixed here: https://github.com/SaraAlert/SaraAlert/pull/793/commits/fb980cdbbffbb305ed63c0cb7a76874d8b185116